metomi/isodatetime/tests/test_00.py

```diff
@@ -159,7 +159,8 @@ def get_timepoint_dumper_tests():
               "+000044-01-04T05:01:02+00:00"),
              ("DD/MM/CCYY is a silly format", "04/01/0044 is a silly format"),
              ("ThhZ", "T05Z"),
-             ("%Y-%m-%dT%H:%M", "0044-01-04T05:01")]
+             ("%Y-%m-%dT%H:%M", "0044-01-04T05:01"),
+             ("T+3", "T+3")]
```

Comment:
@oliver-sanders Any other tests I should add?

Reply:
Would be good to test the CCYYMM type syntax, the example from the issue would suffice.